syne_tune/backend/__init__.py

```diff
@@ -13,6 +13,24 @@
 
 __all__ = ["LocalBackend", "SageMakerBackend", "PythonBackend"]
 
-from syne_tune.backend.local_backend import LocalBackend
-from syne_tune.backend.python_backend.python_backend import PythonBackend
-from syne_tune.backend.sagemaker_backend.sagemaker_backend import SageMakerBackend
+import logging
```

Comment:
David is the expert here. But I believe the dependencies for the backends are in the core.

Of course, we could imagine moving in particular the sagemaker dependencies out of the core.

Reply:
I have to say I still do not understand this. I recently ran random search HPO with the local backend, having installed just the core dependencies (which include SageMaker), and this works fine. So I don't really understand under which circumstances the backend dependencies are not present. Certainly, 'extra' is not needed.